scripts/docs/fixtures/release-notes-legacy-2.4.13/package.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "docs-verifier-legacy-release-notes-fixture",
+  "version": "2.4.13",
```

ESLint

[error] Expected object keys to be in specified order. 'version' should be after 'type'.

(jsonc/sort-keys)